public/index.html

```diff
@@ -0,0 +1,104 @@
+</div>
+<!--end welcome-->
+    
+<div id="listdisplay">
+  <div id="controlsdisplay">
+          <ul>
+
+            <li id="listcount"></li>
+            <li><button type="button" id="search" class="controlbutton">Search</button><input type="text" id="searchInput"></li>
+            <li><label for="autosync">Auto Sync</label>
+          <input type="checkbox" id="autosync"></li>
+            <li><button type="button" id="sync" class="controlbutton">Sync Server</button></li>
+            <li><select id="idDrop">
+              <option>Select Id...</option></select>
+              <a href="#adddisplay"><button type="button" id="updatebutton">Update Record</button></a>
+              <button type="button" id="deletebutton">Delete Record</button><br/></li>
+            <li><button type="button" id="clearlocal">Delete Collection</button></li>
+          </ul>
+    </div>
+
+  <table>
+		<thead>
+          <tr>
+          <td>Artist</td>
+          <td>Title</td>
+          <td>Year</td>  
+          <td>Label</td>
+          <td>SKU</td>
+          <td>Sale</td>
+          <td>Copies</td>
+          <td>Added</td>
+          <td>ID</td>   
+          </tr>
+		</thead>
+    <tbody id="catalogue"> 
+		</tbody>
+	</table>
+</div>
+<!--end listdisplay-->
+    
+<div class="hide" id="adddisplay">
+  <form id="addrecord">
+	<div class="addcol">
+    <label for="artistInput" >Artist</label><br/>
+		<input type="text" id="artistInput" tabindex=1/><br/>
+    <label for="labelInput" >Label</label><br/>
+		<input type="text" id="labelInput" tabindex=4/><br/>
+     <label for="saleInput" >Selling</label>
+      <select id="saleInput" tabindex=7>
+        <option>No</option>
+        <option>Yes</option>
+        </select>
+     </div>
+  
+    
+      <div class="addcol">
+        <label for="titleInput" >Title</label><br/>
+		<input type="text" id="titleInput" tabindex=2 /><br/>
+        <label for="skuInput">SKU</label><br/>
+		<input type="text" id="skuInput" tabindex=5/><br/>
+             <label for="copiesInput">Copies</label><br/>
+		<input type="number" value="1" min="1" id="copiesInput" tabindex=8/>
+            
+   
+      </div>
+    
+  <div class="addcol">
+    <label for="yearInput">Year</label><br/>
+    <select id="yearInput" tabindex=3></select><br/>
+    <label for="idInput">Id</label><br/>
+		<input type="text" id="idInput" tabindex=6/><br/>
+    <button type="button" id="commitnew" tabindex=9>Commit Item</button>
+    <button type="button" id="hidepanel" tabindex=10>Hide Panel</button>
+	</div>
+    </form>
+    <div id="commitdisplay">
+
+  </div>
+</div>
+
+   <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.3/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.12.0/lodash.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/backbone.js/1.3.3/backbone-min.js"></script>
+    <script src="../client.js"></script>
```

Comment:
if you want your server to serve up static files like this one, they'll need to go into the public folder as this is where your static middleware is looking for them